HHS/OPRE-OPS
# Removes the word "Project" before "description"

## What changed

Removes the word "Project" before "description" in Project summary box.

## Issue

#1018 

## How to test

Select a project from the create budget lines / agreement work flow and see the Project Summary Box.
